testsuite/MDAnalysisTests/topology/test_fetch_pdb.py

```diff
@@ -73,49 +101,49 @@ def test_files_to_universe(self, tmp_path, pdb_id, n_atoms):
         assert isinstance(u, mda.Universe) and (len(u.atoms) == n_atoms)
 
 
-@pytest.mark.skipif(not HAS_POOCH, reason="Pooch is not installed.")
-@pytest.mark.skipif(
-    not HAS_ACCESS_TO_WWPDB,
-    reason="Can not connect to https://files.wwpdb.org/",
-)
-class TestExpectedErrors:
-
-    def test_invalid_pdb(self, tmp_path):
-        with pytest.raises(HTTPError):
-            mda.fetch_pdb(pdb_ids="foobar", cache_path=tmp_path)
-
-    def test_invalid_file_format(self, tmp_path):
-        with pytest.raises(ValueError):
-            mda.fetch_pdb(
-                pdb_ids="1AKE", cache_path=tmp_path, file_format="barfoo"
-            )
+@pytest.fixture()
+def clean_up_default_cache():
+    yield
+    rmtree(os_cache("MDAnalysis_pdbs"))
 
 
 @pytest.mark.skipif(not HAS_POOCH, reason="Pooch is not installed.")
 @pytest.mark.skipif(
     not HAS_ACCESS_TO_WWPDB,
     reason="Can not connect to https://files.wwpdb.org/",
 )
-class TestExpectedBehavior:
+class TestExpectedBehaviors:
 
-    def test_no_cache_path(pdb_id):
-        assert isinstance(mda.fetch_pdb('1AKE', cache_path=None), str)
+    def test_no_cache_path(self, clean_up_default_cache):
+        assert isinstance(mda.fetch_pdb("1AKE", cache_path=None), str)
 
-    def test_str_input_gives_str_output(tmp_path):
+    def test_str_input_gives_str_output(self, tmp_path):
         assert isinstance(
-            mda.fetch_pdb(pdb_ids='1AKE', cache_path=tmp_path), str
+            mda.fetch_pdb(
+                pdb_ids="1AKE", cache_path=tmp_path, file_format="cif"
+            ),
+            str,
         )
-    
-    def test_list_input_gives_list_output(tmp_path):
+
+    def test_list_input_gives_list_output(self, tmp_path):
         assert isinstance(
-            mda.fetch_pdb(pdb_ids=['1AKE'], cache_path=tmp_path), list
+            mda.fetch_pdb(pdb_ids=["1AKE"], cache_path=tmp_path), list
         )
 
 
+@pytest.mark.skipif(not HAS_POOCH, reason="Pooch is not installed.")
 @pytest.mark.skipif(
-    HAS_POOCH,
-    reason="Pooch is installed.",
+    not HAS_ACCESS_TO_WWPDB,
+    reason="Can not connect to https://files.wwpdb.org/",
 )
-def test_pooch_installation(tmp_path):
-    with pytest.raises(ModuleNotFoundError):
-        mda.fetch_pdb("1AKE", cache_path=tmp_path, file_format="cif")
+class TestExpectedErrors:
+
+    def test_invalid_pdb(self, tmp_path):
+        with pytest.raises(HTTPError):
```

Comment:
match="..." error output (when you test an exception you want to make sure it is "your" exception)